mergify_engine/delayed_refresh.py

```diff
@@ -53,7 +53,10 @@ async def plan_next_refresh(
     for rule in _rules:
         f = filter.NearDatetimeFilter(rule.conditions.extract_raw_filter_tree())
         live_resolvers.configure_filter(ctxt.repository, f)
-        bet = await f(pull_request)
+        try:
+            bet = await f(pull_request)
```

Comment:
If I read correctly `bet` can be undefined.

Reply:
Hum, we don't use `bet` if the exception is raised or outside the loop, so we are good.

Reply:
right!